yt/yt/client/table_client/wire_protocol.cpp

```diff
@@ -859,13 +862,13 @@ class TWireProtocolReader
     {
         ui32 limit = 0;
         if (type == EValueType::String) {
-            limit = MaxStringValueLength;
+            limit = Limits_.MaxStringValueLength;
         }
         if (type == EValueType::Any) {
-            limit = MaxAnyValueLength;
+            limit = Limits_.MaxAnyValueLength;
         }
         if (type == EValueType::Composite) {
-            limit = MaxCompositeValueLength;
+            limit = Limits_.MaxCompositeValueLength;
         }
         if (length > limit) {
             THROW_ERROR_EXCEPTION("Value of type %Qlv is too long: length %v, limit %v",
```

Comment:
Can we improve this error while we are here? It would be helpful, if we could print some information about the position of the value we are reading within the row. Is it possible?

Reply:
I believe adding a value index to the error is not that hard, but it will require more code changes than in this PR and I am not sure that it is a useful information

I would try to avoid doing it in the same PR

Reply:
Ok.